### ISSUE TITLE
Phone App, No Sound with built in SIP client in Secondary Profile

### ISSUE THREAD

**Issue by southwestgit** [GrapheneOS/os-issue-tracker#550]
3a running latest version. 
RQ2A.210405.005.2021.04.16.04

To Reproduce: Make a SIP call on WiFi in the main profile using the built in Phone App, confirm it works. Call completes and you can hear the other party. 

Switch to a secondary profile and try the same number, call connects but no sound from the other end.  

Additional Info: Linphone works in a secondary profile for the same SIP account to the same phone number. 
However, I am attempting to limit my apps by using built in functionality.

**Comment by thestinger** [comment]
Please try again on the latest June release.

**Comment by southwestgit** [comment]
On RQ3A.210605.005.2021.06.09.13
I can no longer setup the account in the Phone app. 'Invalid Username or Password' I triple checked it and am able to successfully log in elsewhere with the same information. 
Linphone does not like the ~ in the username (not created by me) so this may be an issue for the Phone app as well.

**Comment by luticksi** [comment]
I'm having the same issue with the audio for SIP calls not working on a secondary profile via the Phone app. Owner profile it all works fine. Any updates on this issue?

**Comment by thestinger** [comment]
If the issue had been resolved it would be closed already. Posting replies won't make it into a priority. You can use a different SIP app.

**Comment by luticksi** [comment]
Right, no problem. Just wanted to check in since you mentioned trying again after the June release.

**Comment by vans163** [comment]
Same problem.
